<ui version="4.0" >
 <class>edit_retoptooldialog</class>
 <widget class="QWidget" name="edit_retoptooldialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QToolBox" name="toolBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>441</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="currentIndex" >
    <number>2</number>
   </property>
   <widget class="QWidget" name="pageVertexSelect" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>314</height>
     </rect>
    </property>
    <attribute name="label" >
     <string>1) Vertex selection</string>
    </attribute>
    <widget class="QTableWidget" name="Vtable" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximumSize" >
      <size>
       <width>471</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="columnCount" >
      <number>4</number>
     </property>
     <column>
      <property name="text" >
       <string>Vertex Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>x</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>y</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>z</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="ButtonSelectVertex" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Existing vertex selection&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../fgt/editpaint/meshlab.qrc" >
       <normaloff>:/images/paintbrush-22.png</normaloff>:/images/paintbrush-22.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <property name="autoExclusive" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDeSelectVertex" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>10</y>
       <width>39</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Selected vertex eraser&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../editrgbtri/editrgbtri.qrc" >
       <normaloff>:/images/eraser.png</normaloff>:/images/eraser.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoExclusive" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonSelectVertexFree" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Free vertex selection&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../fgt/editpaint/meshlab.qrc" >
       <normaloff>:/images/paintbrush-22.png</normaloff>:/images/paintbrush-22.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <property name="autoExclusive" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageVertexConnect" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>314</height>
     </rect>
    </property>
    <attribute name="label" >
     <string>2) Vertex connection</string>
    </attribute>
    <widget class="QToolButton" name="ButtonConnectVertex" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Edge creation by vertex selection&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../fgt/editpaint/meshlab.qrc" >
       <normaloff>:/images/paintbrush-22.png</normaloff>:/images/paintbrush-22.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <property name="autoExclusive" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDeConnectVertex" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>10</y>
       <width>39</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Delete edge&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../editrgbtri/editrgbtri.qrc" >
       <normaloff>:/images/eraser.png</normaloff>:/images/eraser.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoExclusive" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="Etable" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximumSize" >
      <size>
       <width>471</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>1st vertex</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>2nd vertex</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pageFaceSelect" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>314</height>
     </rect>
    </property>
    <attribute name="label" >
     <string>3) Face selection and Mesh generation</string>
    </attribute>
    <widget class="QTableWidget" name="Ftable" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>90</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
     <property name="maximumSize" >
      <size>
       <width>471</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="columnCount" >
      <number>3</number>
     </property>
     <column>
      <property name="text" >
       <string>1st vertex</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>2nd vertex</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>3rd vertex</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>120</x>
       <y>0</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title" >
      <string> Retopology mesh generation </string>
     </property>
     <widget class="QPushButton" name="ButtonMeshCreate" >
      <property name="geometry" >
       <rect>
        <x>250</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Create</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSimple" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Simple mesh</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonRef" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Refined mesh, with iterations: </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxTrColor" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Transfer color</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="TextEditIterations" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title" >
      <string> Face selection </string>
     </property>
     <widget class="QToolButton" name="ButtonSelectFace" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Edge creation by vertex selection&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../../fgt/editpaint/meshlab.qrc" >
        <normaloff>:/images/paintbrush-22.png</normaloff>:/images/paintbrush-22.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
      <property name="autoExclusive" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="ButtonSelectFace_2" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Edge creation by vertex selection&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../../fgt/editpaint/meshlab.qrc" >
        <normaloff>:/images/paintbrush-22.png</normaloff>:/images/paintbrush-22.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
      <property name="autoExclusive" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>280</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-size:10pt; color:#ff0000;">Follow the three steps below&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../fgt/editpaint/meshlab.qrc" />
  <include location="../editrgbtri/editrgbtri.qrc" />
 </resources>
 <connections/>
</ui>
